caktus_primer/templates/styleguide/styleguide-modals.html

```diff
@@ -0,0 +1,220 @@
+{% extends "styleguide/styleguide.html" %}
+
+{% load styleguide_tags %}
+<head>
+    {% block extra-css %}
+    <style>
+        dialog {
+            position: relative;
+            padding: 2em 2em;
+            border: none;
+            box-shadow: 2px 3px 4px rgba(0, 0, 0, 0.1);
+        }
+
+        dialog::backdrop {
+            background: #2B2B28;
+            opacity: 0.7;
+        }
```

Comment:
This styles need to actually be in the primer `styl` files, and don't belong embedded in the template.

Reply:
This is not working for me.